src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,213 @@
+    {% comment  %} Add link to the example for this on the create logical replication stream page once published. {% endcomment %}
+
+## Step 2. Connect from the destination to the source
+
+In this step, you'll set up an external connection from the destination cluster to the source cluster. Depending on how you manage certificates, you must ensure that all nodes between the clusters have access to the certificate of the other cluster.
+
+You can use the `cockroach encode-uri` command to generate a connection string containing a cluster's certificate.
+
+1. On the **source** cluster in a new terminal window, generate a connection string, by passing the replication user, node IP, and port, along with the directory to the source cluster's CA certificate:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach encode-uri {replication user}:{password}@{node IP}:26257 --ca-cert {path to CA certificate} --inline
+    ~~~
+
+    The connection string output contains the source cluster's certificate:
+
+    ~~~
+    {replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A
+    ~~~
+
+1. In the SQL shell on the **destination** cluster, create an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) using the source cluster's connection string. Prefix the `postgresql://` scheme to the connection string and replace `{source}` with your external connection name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION {source} AS 'postgresql://{replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A`;
+    ~~~
+
+## Step 3. Start LDR
+
+In this step, you'll start the LDR job from the destination cluster. You can replicate one or multiple tables in a single LDR job. You cannot replicate system tables in LDR, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
+
+_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
+
+- `immediate` (default): Attempts to replicate the changed row directly into the destination table, without re-running constraint validations or triggers. It does not support writing into tables with [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) constraints.
+- `validated`: Attempts to apply the write in a similar way to a user-run query, which would re-run all constraint validations or triggers relevant to the destination table(s). It will potentially reject the change if it fails rather than writing it, which will cause the row change to enter the DLQ instead.
```

Comment:
> It will potentially reject the change if it fails rather than writing it, which will cause the row change to enter the DLQ instead.

I think we should change this sentence to "If the change violates foreign key dependencies, unique constraints, or other constraints, the row will be put in the DLQ instead", because the row will stil be written -- it'll just be written to the DLQ. 

also for this section -- I'm still waiting on eng to provide us with some sample queries the customer can use to get their stuff out of the DLQ. But maybe that would go in the "manage" section

Reply:
Thanks for checking on the queries. And I have changed the language per your rec.